content/docs/capabilities/tcp/client.mdx

```diff
@@ -22,21 +22,32 @@ import LongLivedConnections from '@site/content/docs/admonitions/_long-lived-con
 
 # Pomerium Desktop and CLI Clients
 
-Pomerium is capable of creating secure connections to services like SSH, Redis, and more by creating a TCP tunnel to the service with a local client. This article describes configuring a route to accept TCP connections, and using either the CLI or GUI client to connect to it.
+This document describes how to use Pomerium's Desktop and CLI clients to connect to TCP routes in Pomerium.
 
-## Create a TCP Route
+:::note TCP Terminology
 
-1. Specify this new Route as a TCP Route by prefixing `tcp+` in the **From** field, along with a port suffix.
+In this document, a "TCP route" refers to a route configured to accept TCP connections.
 
-   The port is not used to connect to the Pomerium Proxy service from the internet; this will always be port 443 (unless otherwise defined in `config.yaml`). Rather, the port defined in **From** is part of the mapping to the individual route. In this way, you can create multiple routes that share a DNS entry, differentiated by the port to determine which route they use.
+:::
 
-   For example, suppose we have a server called `augur` running behind Pomerium that has a MySQL server and also listens for SSH connections. We can create routes for `tcp+https://augur.example.com:22` and `tcp+https://augur.example.com:3306`.
+## Create a TCP route
 
-1. The **To** field uses `tcp://` as a protocol, and specifies the address and port the service listens on.
+1. Specify this new Route as a TCP Route by prefixing `tcp+` in the **From** field, along with a port suffix
+1. The **To** field uses `tcp://` as a protocol, and specifies the address and port the service listens on
```

Comment:
While we're here, I think it would make sense to rephrase this step as an instruction, for consistency with step 1.

```suggestion
1. Set the **To** field using `tcp://` as the scheme, along with the address and port that the upstream service is listening on.
```

Reply:
Nice catch, and I agree.